Testscripts/Linux/KDUMP-Execute.sh

```diff
@@ -149,13 +149,17 @@ Exec_mariner()
 
 Configure_NMI()
 {
-    sysctl -w kernel.unknown_nmi_panic=1
-    if [ $? -ne 0 ]; then
-        LogErr "Failed to enable kernel to call panic when it receives a NMI."
-        SetTestStateAborted
-        exit 0
+    if [ -e "/proc/sys/kernel/unknown_nmi_panic" ] ; then
+        sysctl -w kernel.unknown_nmi_panic=1
+        if [ $? -ne 0 ]; then
+            LogErr "Failed to enable kernel to call panic when it receives a NMI."
+            SetTestStateAborted
+            exit 0
+        else
+            UpdateSummary "Success: enabling kernel to call panic when it receives a NMI."
+        fi
     else
-        UpdateSummary "Success: enabling kernel to call panic when it receives a NMI."
+        UpdateSummary "The kernel doesn't support to call panic when it receives a NMI."
```

Comment:
Should we set test result skip here and do you know what cause this path doesn't exist? /proc/sys/kernel/unknown_nmi_panic

Reply:
Some architectures don't provide architected NMIs,such as ARM64. So this file doesn't exist. I added comments. This is configured for Hyper-V testing. We can just ignore it and keep running. It won't affect kdump.